Runtimes/Core/runtime/CMakeLists.txt

```diff
@@ -94,14 +108,14 @@ target_compile_definitions(swiftRuntime
     $<$<BOOL:${SwiftCore_ENABLE_OS_TRACE_LAZY_INIT}>:-DSWIFT_USE_OS_TRACE_LAZY_INIT>)
 
 target_include_directories(swiftRuntime PRIVATE
-  "${PROJECT_BINARY_DIR}/include"
   "${CMAKE_CURRENT_SOURCE_DIR}")
 
 target_link_libraries(swiftRuntime PRIVATE
   $<$<PLATFORM_ID:Windows>:User32>
   $<$<PLATFORM_ID:Windows>:ShLwApi>
   swiftShims
-  swiftDemangling)
+  swiftDemangling
+  swiftRuntimeCMakeConfig)
```

Comment:
I don't think that we should be doing this part of the change - we should restore the include path for the generated config.